app/controllers/csv_preview_controller.rb

```diff
@@ -58,6 +58,10 @@ def asset_path
     request.path.sub("/preview", "").sub(/^\//, "")
   end
 
+  def filename
```

Comment:
Nitpick, but I'd probably call this method asset_filename to be consistent with asset_path

Reply:
done